block.go

```diff
@@ -95,6 +96,7 @@ type BlockMeta struct {
 }
 
 const (
+	// XXX: What is the meaning of these flags?
```

Comment:
Original serialization format had a lot more versioning flags. As there was only the first version of everything I just added these standard constants.

Ultimately, I cut down on a fair bit of complexity and most version flags in #61.
So those can soon be removed.